build/root/Makefile.generated_files

```diff
@@ -205,7 +206,8 @@ $(PRERELEASE_LIFECYCLE_FILES): $(PRERELEASE_LIFECYCLE_GEN)
 # newer than the binary, and try to "rebuild" it over and over.  So we touch
 # it, and make is happy.
 $(PRERELEASE_LIFECYCLE_GEN): $(GODEPS_k8s.io/kubernetes/vendor/k8s.io/code-generator/cmd/prerelease-lifecycle-gen)
-	KUBE_BUILD_PLATFORMS="" hack/make-rules/build.sh ./vendor/k8s.io/code-generator/cmd/prerelease-lifecycle-gen
+	KUBE_BUILD_PLATFORMS="" \
+	    hack/make-rules/build.sh vendor/k8s.io/code-generator/cmd/prerelease-lifecycle-gen
```

Comment:
huh, leading `./` got dropped?

Reply:
oh, this isn't go install, it was already calling build.sh

Reply:
yeah, the ./ just shows up in the output in a weird way, and isn't strictly needed.  And actually, later in my go-worspaces tree, I think it changes again.  I could revert this part if you care.

Reply:
oh, that part already merged in a different PR.  Still, I can undo it if it bothers :)